packages/compat/source/adapter/host.ts

```diff
@@ -251,11 +251,23 @@ function adaptLegacyNodeSerialization(
 ): RemoteElementSerialization | RemoteTextSerialization {
   if (child.kind === LEGACY_KIND_TEXT) {
     return adaptLegacyTextSerialization(child);
+  } else if (child.kind === LEGACY_KIND_FRAGMENT) {
+    return adaptUndefinedLegacyFragmentSerialization(child);
   } else {
     return adaptLegacyComponentSerialization(child, options);
   }
 }
 
+function adaptUndefinedLegacyFragmentSerialization(props: any) {
```

Comment:
Remote-ui does the same 

<img width="928" alt="Screenshot 2025-03-03 at 3 51 12 PM" src="https://github.com/user-attachments/assets/a18961b5-14e9-4fc2-8fac-4d6e79a7cca5" />

Remote-dom with this cha

<img width="913" alt="Screenshot 2025-03-03 at 3 55 19 PM" src="https://github.com/user-attachments/assets/9303b392-c769-4f3e-bc7c-71eab86e0df1" />
nge